app/src/features/surveys/observations/sampling-sites/components/SamplingSiteMapControl.tsx

```diff
@@ -120,8 +120,8 @@ const SamplingSiteMapControl = (props: ISamplingSiteMapControlProps) => {
           <Typography component="legend" data-testid="funding-source-list-found">
             Site Location Preview &zwnj;
             <Typography component="span" color="textSecondary" fontWeight="400">
```

Comment:
I would conditionally render this `<Typography>` rather than render it with an empty string